x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_update_rules/route.ts

```diff
@@ -69,51 +71,86 @@ export const bulkUpdateRulesRoute = (
         savedObjectsClient,
       });
 
+      const ruleDefinitions = request.body;
+
+      const rulesIndexesWithDuplicatedDefaultExceptionsList =
+        getRulesIndexesWithDuplicatedDefaultExceptionsList(ruleDefinitions);
+
       const rules = await Promise.all(
-        request.body.map(async (payloadRule) => {
-          const idOrRuleIdOrUnknown = payloadRule.id ?? payloadRule.rule_id ?? '(unknown id)';
-          try {
-            const validationErrors = validateUpdateRuleProps(payloadRule);
-            if (validationErrors.length) {
-              return createBulkErrorObject({
+        ruleDefinitions
+          .filter((rule, index) => !rulesIndexesWithDuplicatedDefaultExceptionsList.includes(index))
```

Comment:
Not sure I get this `filter`. Are we getting rid of rules from the payload that have duplicated default exceptions list, and then moving forward with the `update` with the others?

In that case, shouldn't we let the user of the API know that those rules were excepted or skipped from the update, instead of "failing silently"?

Reply:
In bellow, you can see that we add errors to `rulesBulk` variable/

It's similar as we do with duplication in bulk create